src/app/static/src/app/components/outputTable/TableData.vue

```diff
@@ -1,3 +1,65 @@
 <template>
-    <div>Table</div>
-</template>
\ No newline at end of file
+    <table-display :data="filteredData"/>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from 'vue';
+import { RootState } from '../../root';
+import { useStore } from 'vuex';
+import { Filter } from '../../generated';
+import TableDisplay from './TableDisplay.vue';
+
+export default defineComponent({
+    components: {
+        TableDisplay
+    },
+    setup() {
+        const store = useStore<RootState>();
+        const selections = computed(() => store.state.plottingSelections.table);
+        // this is necessary because filters are called
+        // "age" for example on the filter selections when
+        // the id in the data array is "age_group"
+        const filtersIdToDataId = computed(() => {
+            const filters: Filter[] = store.getters["modelOutput/tableFilters"] || [];
+            const idToDataId: Record<string, string> = {};
+            filters.forEach(f => {
+                idToDataId[f.id] = f.column_id;
+            });
+            return idToDataId;
+        });
+
+        const filteredData = computed(() => {
+            const result = store.state.modelCalibrate.result;
+            if (!result) {
+                return []
+            } else {
+                const data = result.data;
+                const selectedOptions = selections.value.selectedFilterOptions;
+                const filterKeys = Object.keys(selectedOptions);
+                const selectedOptionIds: Record<string, (string | number)[]> = {};
+                for (const k in selectedOptions) {
+                    if (k === "area_level") {
+                        selectedOptionIds[k] = selectedOptions[k].map(op => parseInt(op.id));
```

Comment:
That's a shame!
I guess the area level values are in the data as numbers, so we do need to deal with them as numbers, but it's pretty gross needing to have a baked in special that the component is aware of, given that we're pretty metadata-driven other than this. 
Could have a valueType in the metadata I guess (which is also a bit eurgh) Or could actually get these as strings from the database, if we're not using it for anything else.

Reply:
I could also return the filter option values as integers if that helps. Though perhaps that would be a bigger change as I think they're always string at the moment

Reply:
Worth adding a test which will check this path too?

Reply:
I think it might be a good idea to return detail level as a string actually to the frontend and put like one exception maybe in the backend? not right now but potentially later on when we do the plots cleanup epic?

Reply:
will add a test after standup

Reply:
done